corehq/form_processor/models.py

```diff
@@ -208,6 +208,12 @@ class XFormInstanceSQL(PartitionedModel, models.Model, RedisLockableMixIn, Attac
     # for compatability with corehq.blobs.mixin.DeferredBlobMixin interface
     persistent_blobs = None
 
+    # form meta properties
+    time_end = models.DateTimeField(null=True, blank=True)
+    time_start = models.DateTimeField(null=True, blank=True)
+    commcare_version = models.CharField(max_length=8, blank=True, null=True)
+    build_version = models.PositiveSmallIntegerField(null=True, blank=True)
```

Comment:
Err. I think I wasn't super clear on my previous comment. I think this might overflow at some point? The max is 32k for small integers. ICDS is currently > 10k. Not sure if we want to take that chance? Maybe `SMallIntegerField` would be better?

Reply:
+1 re overflow though `SmallIntegerField` has the same upper bound. The next option is `PositiveIntegerField` (same size as `IntergerField` but restricts to values >= 0)

Reply:
Cool, fixed it